source/classes.tex

```diff
@@ -6785,10 +6785,76 @@
 \rSec2[class.spaceship]{Three-way comparison}
 \indextext{operator!three-way comparison!defaulted}%
 
+\pnum
+The synthesized three-way comparison
+for comparison category type \tcode{R}\iref{cmp.categories}
+of glvalues \tcode{a} and \tcode{b} of the same type
+is defined as follows:
+
+\begin{itemize}
+\item
+If overload resolution for \tcode{a <=> b}
+finds a usable function \iref{over.match},
+\tcode{static_cast<R>(a <=> b)}.
+
+\item
+Otherwise, if overload resolution for \tcode{a <=> b}
+finds at least one viable candidate,
+the synthesized three-way comparison is not defined.
+
+\item
+Otherwise, if \tcode{R} is \tcode{strong_ordering}, then
+\begin{codeblock}
+a == b ? strong_ordering::equal :
+a < b  ? strong_ordering::less :
+         strong_ordering::greater
+\end{codeblock}
+
+\item
+Otherwise, if \tcode{R} is \tcode{weak_ordering}, then
+\begin{codeblock}
+a == b ? weak_ordering::equal :
+a < b  ? weak_ordering::less :
+         weak_ordering::greater
+\end{codeblock}
+
+\item
+Otherwise, if \tcode{R} is \tcode{partial_ordering}, then
+\begin{codeblock}
+a == b ? partial_ordering::equivalent :
+a < b  ? partial_ordering::less :
+b < a  ? partial_ordering::greater :
+         partial_ordering::unordered
+\end{codeblock}
+
+\item
+Otherwise, if \tcode{R} is \tcode{strong_equality}, then
+\begin{codeblock}
+a == b ? strong_equality::equal : strong_equality::nonequal
```

Comment:
Suggestion: place `strong_equality::nonequal;` on new line for consistency with the previous items?

Reply:
Tried it, it looked a bit weird. We often only flow things onto multiple lines if they don't fit on one line; I think this case is similar.